test/manual/amp-ad.yieldbot.amp.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>AMP-AD.yieldbot</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async custom-element="amp-iframe" src="../../dist/v0/amp-iframe-0.1.max.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="../../dist/amp.js"></script>
+</head>
+<body>
+    <!-- Sets i.yldbt.com integration testing cookie __ybot_test -->
+    <!-- 1x1 transparent gif placeholder -->
+    <amp-iframe width=1 height=1
+                sandbox="allow-scripts allow-same-origin"
+                frameborder="0"
+                src="https://i.yldbt.com/m/start-testing">
+      <amp-img layout="fill" src="data:image/gif;base64,R0lGODlhAQABAIAAAP///////yH5BAEKAAEALAAAAAABAAEAAAICTAEAOw==" placeholder></amp-img>
+  </amp-iframe>
+
+  <h2>Yieldbot 300x250</h2>
+  <amp-ad width="300" height="250"
+          type="yieldbot"
+          data-psn="1234"
+          data-yb-slot="medrec"
+          data-slot="/2476204/medium-rectangle"
+          json='{"targeting":{"category":["food","lifestyle"]},"categoryExclusions":["health"]}'>
+
+  </amp-ad>
+
+  <!-- Unset i.yldbt.com integration testing cookie __ybot_test -->
+  <!-- 1x1 transparent gif placeholder -->
+  <amp-iframe width=1 height=1
```

Comment:
Will need to remove this b/c the second `amp-iframe` is detected as a tracking pixel it seems.

Will likely need to put an href in the test page in some way such that the integration testing mode can be disabled. Since the integration testing is enabled via a cookie on i.yldbt.com, it's possible that a user may have the testing mode inadvertently in an enabled state and may not realize how to disable the testing mode.